public class Exercicio3_Exceptions {
    public static void main(String[] args) {

        CalculoMatematico resultado = new CalculoMatematico();

        resultado.divisao( 4, 0);

    }
}
